README.md

```diff
@@ -1,6 +1,46 @@
-# Static recomp of IRIX programs
+# Static Recompilation of IRIX Programs
+Convert selected IRIX C toolchain programs into modern Linux or macOS programs
 
-Example for compiling `as1`:
+## Supported Programs
+* IDO 5.3
+  * cc, acpp, as0, as1, cfe, copt, ugen, ujoin, uld, umerge, uopt, usplit
+* IDO 7.1
+  * cc, as1, cfe, ugen, umerge, uopt
+
+# Dependencies
+Note that the `python3` dependency is only necessary if building with the `build.py` script. 
+Likewise, the `make` dependency is only needed if building with `make`.
+
+## Linux (Debian / Ubuntu)
+```bash
+sudo apt-get install build-essential libcapstone-dev pkg-config python3
+```
+
+## macOS
+[Install homebrew](https://brew.sh/) and then:
+```bash
+brew install make python3 pkg-config capstone
+```
+Use `gmake` instead of `make` in order to use homebrew's `make`
+
+## Build using Make
+```bash
+make VERSION=5.3
+make VERSION=7.1
+```
+The build artifacts are located in `build/{IDO71|IDO53}/bin`
```

Comment:
Is there a particular reason that these are different?

Reply:
Do you mean different output directories for each IDO version, or do you mean why are there different build directories for the python script vs the Makefile?

Reply:
Yeah, I mean why make uses bin and the python script out.

Reply:
I was writing and didn't notice haha. I could easily change the makefile output directory to be `out`, but maybe I should reconcile all of the differences between the Makefile and `build.py`. What do you think?

Reply:
That seems sensible to me.